OHHTTPStubs/Pods/AFNetworking/AFNetworking/AFNetworkReachabilityManager.h

```diff
@@ -75,7 +75,7 @@ NS_ASSUME_NONNULL_BEGIN
 
 /**
  Creates and returns a network reachability manager with the default socket address.
- 
+
```

Comment:
Careful, we shouldn't modify the code of third-party pods 😉

Maybe re-run `pod install` after your regex to overwrite the pods's code back to their original code for example